### PR TITLE
New Keyboard: Doro67 Regular PCB

### PR DESCRIPTION
**Pull request by mechmerlin**
## Description
Added in the regular PCB for the Doro67. This is the PCB currently in customers hands. RGB PCB coming soon. 

## Types of changes

- [ ] Core
- [ ] Bugfix
- [ ] New Feature
- [ ] Enhancement/Optimization
- [x] Keyboard (addition or update)
- [x] Keymap/Layout/Userspace (addition or update)
- [ ] Documentation

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document. (https://docs.qmk.fm/#/contributing)
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/doro67/regular/keymaps/default/config.h [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/* Copyright 2019 'mechmerlin'
```
Comment (noroadsleft) [inline keyboards/doro67/regular/keymaps/default/config.h]:
```suggestion
/* Copyright 2019 MechMerlin
```

File: keyboards/doro67/regular/readme.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Regular PCB for the Doro67 
+
+Regular PCB for the Doro67 having ONLY one layout. This is not the PCB with RGB support. 
+
+Do not flash RGB firmware for this board. 
+
+Keyboard Maintainer: [MechMerlin](https://github.com/mechmerlin)  
+Hardware Supported: Doro67
```
Comment (noroadsleft) [inline keyboards/doro67/regular/readme.md]:
```suggestion
Hardware Supported: Doro67 Regular PCB  
```

File: keyboards/doro67/regular/regular.h [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+/* This a shortcut to help you visually see your layout.
+ *
+ * The first section contains all of the arguments representing the physical
+ * layout of the board and position of the keys.
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ */
+#define LAYOUT( \
+  K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K0C, K0D, K0E, \
```
Comment (noroadsleft) [inline keyboards/doro67/regular/regular.h]: Someday I'll figure out how to get the KBFirmware JSON parser tool to auto-white-space this section.

**Comment by drashna** [comment]
Thanks!

**Pull request merged**